libjsonutils/cmake/JSONUtilsConfig.cmake.in

```diff
@@ -1,18 +1,18 @@
-get_filename_component(JSONUtils_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-include(CMakeFindDependencyMacro)
-
-list(APPEND CMAKE_MODULE_PATH ${JSONUtils_CMAKE_DIR})
-
-# NOTE Had to use find_package because find_dependency does not support COMPONENTS or MODULE until 3.8.0
-
-#find_dependency(Boost 1.55 REQUIRED COMPONENTS regex)
-#find_dependency(RapidJSON 1.0 REQUIRED MODULE)
-find_package(Boost 1.55 REQUIRED COMPONENTS regex)
-find_package(RapidJSON 1.0 REQUIRED MODULE)
+get_filename_component(JSONUTILS_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+#
+# NOTE: We had to use find_package because find_dependency does not support
+# COMPONENTS or MODULE until 3.15.0? CK
+#
+list(APPEND CMAKE_MODULE_PATH ${JSONUTILS_CMAKE_DIR})
+# NOTE: to find FindRapidJSON.cmake
+find_package(RAPIDJSON @MIN_RAPIDJSON_VERSION@ REQUIRED MODULE)
 list(REMOVE_AT CMAKE_MODULE_PATH -1)
 
+include(CMakeFindDependencyMacro)
+find_dependency(boost_regex @MIN_BOOST_VERSION@ REQUIRED CONFIG)
+
 if(NOT TARGET JSONUtils::JSONUtils)
-    include("${JSONUtils_CMAKE_DIR}/JSONUtilsTargets.cmake")
+    include("${JSONUTILS_CMAKE_DIR}/JSONUtilsTargets.cmake")
 endif()
 
-set(JSONUtils_LIBRARIES JSONUtils::JSONUtils)
+set(JSONUTILS_lIBRARIES JSONUtils::JSONUtils)
```

Comment:
It is far better to match the case with the module name. This should also only be done if the module is found (basically, the dependencies exist).

Reply:
OK